statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -2711,15 +2719,15 @@ def summary(self, alpha=.05, start=None, title=None, model_name=None,
         # Diagnostic tests results
         try:
             het = self.test_heteroskedasticity(method='breakvar')
-        except:
+        except Exception:  # FIXME: catch something specific
             het = np.array([[np.nan]*2])
         try:
             lb = self.test_serial_correlation(method='ljungbox')
-        except:
+        except Exception:  # FIXME: catch something specific
             lb = np.array([[np.nan]*2]).reshape(1, 2, 1)
         try:
             jb = self.test_normality(method='jarquebera')
-        except:
+        except Exception:  # FIXME: catch something specific
```

Comment:
@ChadFulton This is the next-to-last thing that needs your input.

Reply:
What do you think about just `# noqa` on these? Two reasons:

- These are in the summary, and I think that we want to show the summary regardless of any errors here.
- There are a whole host of exceptions that could arise from the class methods or from the functions that they call, so it might be tough to narrow down all of the possible exceptions.

Reply:
At the very least it should be `exception Exception:`.  With just `except:` we will e.g. catch `KeyboardInterrupt` which is definitely undesired.

I'd be OK with leaving the "FIXME" here for the time being.  It's a low enough priority that we don't need to fix it immediately, but high enough that it should be hard to ignore on the next pass.

Reply:
> At the very least it should be exception Exception:. With just except: we will e.g. catch KeyboardInterrupt which is definitely undesired.

Good point.

> I'd be OK with leaving the "FIXME" here for the time being. It's a low enough priority that we don't need to fix it immediately, but high enough that it should be hard to ignore on the next pass.

Okay, we can discuss later.